src/mono/wasm/runtime/memory.ts

```diff
@@ -408,3 +408,11 @@ export function isSharedArrayBuffer(buffer: any): buffer is SharedArrayBuffer {
     // See also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag
     return sharedArrayBufferDefined && buffer[Symbol.toStringTag] === "SharedArrayBuffer";
 }
+
+export function forceThreadMemoryViewRefresh() {
+    // this condition should be eliminated by rollup on non-threading builds
+    if (!MonoWasmThreads) return false;
```

Comment:
Why false when we never check the returned value?
```suggestion
    if (!MonoWasmThreads) return;
```